ta/pkcs11/src/processing_digest.c

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2021, Linaro Limited
```

Comment:
Wondering what copyright I should put in here?

This is now new file based on other processing files.

Should there be Vaisala copyright too?

Reply:
Depends, if you rewrote a lot of the code you could put it under your own copyright, if you are aggregating code which was already written by Linaro I'd keep the original copyright.

Reply:
If that's Linaro code + yours, then put both :)

Reply:
Note that from my perspective, you don't need to add the Linaro copyright.